test/functional/rpc_rawtransaction.py

```diff
@@ -69,24 +74,129 @@ def setup_network(self):
         super().setup_network()
         self.connect_nodes(0, 2)
 
+    def sync_peers(self):
+        self.sync_mempools(self.nodes[0:3])
```

Comment:
Question: Is this intentionally only on 4 nodes instead of 6? If it is, maybe `sync_peers` should be called slightly differently not to give the wrong impression.

Reply:
> Question: Is this intentionally only on 4 nodes instead of 6?

Yes, as only nodes 0 to 3 have a mempool. The last two nodes are -blocksonly and `sync_mempools()` won't work on them.  Added a comment to clarify this.

```diff
-        self.sync_mempools()
+        self.sync_mempools(self.nodes[0:3])  # nodes 0 to 3 have a mempool
```